packages/opam-compiler/opam-compiler.0.2.1/opam

```diff
@@ -0,0 +1,53 @@
+opam-version: "2.0"
+synopsis: "Plugin to create switches using custom compilers"
+description:
+  "This plugin can manage switches using various sources for compilers, such as git branches, github PRs, etc"
+maintainer: ["Etienne Millon <me@emillon.org>"]
+authors: ["Etienne Millon <me@emillon.org>"]
+license: "ISC"
+homepage: "https://github.com/ocaml-opam/opam-compiler"
+bug-reports: "https://github.com/ocaml-opam/opam-compiler/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.08.0"}
+  "bos"
+  "cmdliner" {>= "1.1.0"}
+  "cmdliner" {>= "1.2.0" & with-test}
+  "curly" {>= "0.2.0"}
+  "either" {with-test}
+  "github-data"
+  "ocaml-version" {>= "3.0.0"}
+  "re" {>= "1.5.0"}
+  "rresult" {>= "0.6.0"}
+  "uri"
+  "alcotest" {>= "1.2.0" & with-test}
+  "odoc" {with-doc}
+]
+conflicts: [
+  "result" {< "1.5"}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/ocaml-opam/opam-compiler.git"
+flags: plugin
```

Comment:
```suggestion
flags: plugin
available: opam-version >= 2.1
```

Fix for the opam.2.0 installation error, I'm not planning on supporting such an old opam version anyway.